app/src/ingest_policy_pdfs.py

```diff
@@ -106,11 +104,13 @@ def process_chunk(text: str, document: Document, db_session: db.Session) -> None
             current_chunk.append(sentence)
             current_token_count += token_count
         else:
+            chunk_text = "".join(current_chunk)
+            chunk_embedding = embedding_model.encode(chunk_text, show_progress_bar=False)
```

Comment:
Note that `encode()` accepts a `List[str]`, so we may not need to `"".join(current_chunk)`.

Reply:
That encodes each str separately and then returns a list of embeddings for each: https://sbert.net/docs/package_reference/sentence_transformer/SentenceTransformer.html#sentence_transformers.SentenceTransformer.encode